module/dateSet.js

```diff
@@ -0,0 +1,21 @@
+export const dateEl = document.querySelector('.date');
+
+const setDate = () => {
+  const now = new Date();
+  const options = {
+    month: 'long',
+    day: '2-digit',
+    year: 'numeric',
+    hour: '2-digit',
+    minute: '2-digit',
+    second: '2-digit',
+    hour12: true,
+  };
+  const date = Intl.DateTimeFormat([], options).format(now);
+  return date;
+};
+
+setInterval(() => {
+  dateEl.textContent = '';
+  dateEl.textContent = setDate();
+});
```

Comment:
- [ ] Based on project requirements, you are supposed to use the [Luxon](https://moment.github.io/luxon/docs/manual/install.html) library for displaying time. Because this is a very handy tool that you will do in your future job.